specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/examples/ManagedEnvironments_CreateOrUpdate.json

```diff
@@ -45,6 +90,10 @@
         "name": "testcontainerenv",
         "type": "Microsoft.App/managedEnvironments",
         "location": "East US",
+        "sku": {
+          "name": "Premium",
+          "capacity": 10
+        },
```

Comment:
How 10 is related to below max and min number

Reply:
This is total nodes available for the environment i.e. sum of workload profile max can be less than equal to cluster maximum